jobs/JGLOBAL_WAVE_POST_SBS

```diff
@@ -9,9 +9,17 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_GRID
-
-mkdir -p "${COM_WAVE_GRID}"
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+    COMIN_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
+    COMIN_WAVE_GRID:COM_WAVE_GRID_TMPL \
+    COMOUT_WAVE_PREP:COM_WAVE_PREP_TMPL \
```

Comment:
I do not think we need COMOUT_WAVE_PREP or COMOUT_WAVE_HISTORY

Reply:
`COMOUT_WAVE_PREP` is required by `ush/wave_grid_interp_sbs.sh`. `COMOUT_WAVE_HISTORY` has been removed.

Reply:
I'm not sure COMOUT_WAVE_PREP should exist either.. that seems like a mistake (perhaps in the code itself).